docs/tutorial-part-2.rst

```diff
@@ -0,0 +1,977 @@
+.. _tutorial:
+
+pg_auto_failover Tutorial: Part II
+==================================
+
+In this guide we'll create a primary and two secondary Postgres nodes in the
+Paris Azure region (`francecentral`). Then because our business requirements
+are evolving, we prepare a new region in Amsterdam (`westeurope`) with
+another set of 3 nodes that we intend to use as a fallback in case the whole
+`francecentral` region would go down.
+
+On-top of those two regions, we also create a lone standby node in the
+Dublin area (`northeurope` Azure region) to have another copy of our data
+set.
+
+Once this setup is all in place, we're going to orchestrate a migration of
+the Postgres production from the Paris region to the Amsterdam region: when
+using pg_auto_failover, that's a simple `switchover` command. Some
+preparation is still necessary before doing the switchover though, we want
+to have the replication quorum and candidate priorities just right.
+
+Create the Azure Network and Connect them
+-----------------------------------------
+
+This time, preparing the Azure components is going to be a long story in
+itself. We need three resource groups, and in each of them we need a gateway
+with its own public IP address. And then we need to connect our regions with
+a VNet-to-VNet VPN service. Finally we'll create the virtual machines,
+install our Postgres and pg_auto_failover packages, and get started with the
+actual failover management parts.
+
+Create virtual network in Paris (francecentral)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Our database machines need to talk to each other and to the monitor node, so
+let's create a virtual network.
+
+.. code-block:: bash
+
+   az group create \
+       --name ha-demo-dim-paris \
```

Comment:
Don't forget to remove the `-dim` part from these strings.

Reply:
I think there is more to it than that, but of course this. Adding a top-level comment with what I have in mind more generally.